### DISCUSSION TITLE
Hidding view action button

### DISCUSSION THREAD

**Discussion by ShaungBhone** [filamentphp/filament#11757] [Help] [answered]
### Package

Infolist builder

### Package Version

v3

### How can we help you?

How can I hide view action button in infolist builder in livewire component? In my table I just want to show only edit button. When user is click the table row that is go to view page. Thank you.

**Comment by leandrocfe** [comment]
Remove view action or use `hidden`

```php
Tables\Actions\ViewAction::make()
    ->hidden()
```

**Reply by ShaungBhone** [reply]
Yes, it was hidden but I can't click the row.

**Reply by ShaungBhone** [reply]
```
Tables\Actions\ViewAction::make()
                    ->hidden()
                    ->url(fn (Business $business): string => route('business.view', ['business' => $business])),
```

**Reply by leandrocfe** [reply, marked as answer]
```php
return $table
    ->recordUrl(...) //or
    ->recordAction(...)
```

**Reply by ShaungBhone** [reply]
Thank you so much.
